### ISSUE TITLE
[HW] <SMANIKA>

### ISSUE THREAD

**Issue by Smanika** [datastaxdevs/appdev-week3-graphql#320]
**Name:**  SMANIKA
**Email:** <smanika2222@gmail.com>

Attach the homework screenshots below:
-----------------------------------------
HOMEWORK DONE!!!

<SCREENSHOTS>
![Screenshot (187)](https://user-images.githubusercontent.com/67970197/126530219-13b13e2c-8cd5-4cef-b985-cff0e53fc587.png)
![Screenshot (188)](https://user-images.githubusercontent.com/67970197/126530228-b034b927-6326-41b1-bdc5-b0ce8fc0472c.png)
![Screenshot (190)](https://user-images.githubusercontent.com/67970197/126530286-b591f83f-5041-446b-a716-9df62079a2ad.png)
![Screenshot (191)](https://user-images.githubusercontent.com/67970197/126530290-113364c2-fef0-4b64-a78e-97a7da4dcf7d.png)
![Screenshot (192)](https://user-images.githubusercontent.com/67970197/126530293-e2fb0004-3946-4bd3-9b3f-a315e505ee9b.png)
![Screenshot (193)](https://user-images.githubusercontent.com/67970197/126530294-c533e49b-4254-483e-803f-1f4abe307766.png)
![Screenshot (194)](https://user-images.githubusercontent.com/67970197/126530298-0e5f683c-0009-4a10-9491-f7e03968df55.png)
![Screenshot (195)](https://user-images.githubusercontent.com/67970197/126530302-0954db26-d9ae-4a03-988e-c720023c6a0b.png)
![Screenshot (196)](https://user-images.githubusercontent.com/67970197/126530308-31a1763f-fa21-40ea-8c47-10bc532d3b53.png)
![Screenshot (197)](https://user-images.githubusercontent.com/67970197/126530313-fb97767e-c2d1-49ea-9711-b0b840bc6b10.png)
![Screenshot (198)](https://user-images.githubusercontent.com/67970197/126530317-53af9eae-4850-4ff6-86d1-f9fa09128044.png)
![Screenshot (199)](https://user-images.githubusercontent.com/67970197/126530321-7baff3b8-1b04-4de7-9de5-fd63cba3e41b.png)
![Screenshot (201)](https://user-images.githubusercontent.com/67970197/126530327-deaa27ff-9b16-4c57-9b90-0de309fb9e4e.png)
![Screenshot (202)](https://user-images.githubusercontent.com/67970197/126530332-3ebe53f2-2b69-4f5f-a879-18444ed9eb4a.png)
![Screenshot (203)](https://user-images.githubusercontent.com/67970197/126530336-0b7968f7-d5c3-4ef2-9115-b69e510ead66.png)
![Screenshot (204)](https://user-images.githubusercontent.com/67970197/126530338-c3d0937b-3ef3-4267-a527-af405d1987d7.png)
![Screenshot (205)](https://user-images.githubusercontent.com/67970197/126530341-41f5f06a-8f6c-4b7c-8c10-97b22628eb29.png)
![Screenshot (206)](https://user-images.githubusercontent.com/67970197/126530343-11142087-9bfa-4742-9abf-b9b35aa8acb7.png)
![Screenshot (207)](https://user-images.githubusercontent.com/67970197/126530348-3f9eb5d9-a596-4b34-a717-7d032e67cfa4.png)
![Screenshot (208)](https://user-images.githubusercontent.com/67970197/126530350-f4dc0a11-689b-454e-a8d9-9e39df5135f2.png)

**Comment by SonicDMG** [comment]
Hi Smanika, nice job getting the app deployed. I don't see your custom video though on your site. Could you please add one and take a screenshot or just post your netlify link. If you did add one please point out which one. Thank you.

**Comment by Smanika** [comment]
https://blissful-mcnulty-287ba6.netlify.app/

**Comment by Smanika** [comment]
Haven't got issued the badge yet. Is there any correction in my work??

**Comment by SonicDMG** [comment]
Yes, which custom video did you add?

**Comment by Smanika** [comment]
I had to add video myself like of my own choice??

**Comment by SonicDMG** [comment]
> I had to add video myself like of my own choice??

Yes, that is correct. Per step 2 of the homework "Insert a movie of your choice in the database." You can go back to [step 7 of the instructions](https://github.com/datastaxdevs/appdev-week3-graphql#7-insert-values-in-movie-table) and modify one of the mutations to add your own custom video.
